doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -72,7 +72,7 @@ In order for the Kubernetes components to be conformant with the Reference Archi
 
 |Ref|Specification|Details|Requirement Trace|Reference Implementation Trace|
 |---|---|---|---|---|
-|`ra2.k8s.001`|Kubernetes Conformance|The Kubernetes distribution, product, or installer used in the implementation **must** be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) and marked (X) as conformant for Kubernetes version 1.21.|[req.gen.cnt.03](./chapter02.md#23-kubernetes-architecture-requirements)||
+|`ra2.k8s.001`|Kubernetes Conformance|The Kubernetes distribution, product, or installer used in the implementation **must** be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) and marked (X) as conformant for the Kubernetes version defined in `ra2.k8s.005`.|[req.gen.cnt.03](./chapter02.md#23-kubernetes-architecture-requirements)||
```

Comment:
Suggestion instead of referring to `ra2.k8s.005` can we refer to the README? Also change it in `ra2.k8s.005`.
```suggestion
|`ra2.k8s.001`|Kubernetes Conformance|The Kubernetes distribution, product, or installer used in the implementation **must** be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) and marked (X) as conformant for the Kubernetes version defined in [README](./README.md) under "Required versions of most important components" .|[req.gen.cnt.03](./chapter02.md#23-kubernetes-architecture-requirements)||
```

Reply:
I somehow consider listing the version(s) in the README as a conviniency feature and not as the canonical source of information. I would rather prefer to remove the list from the README and keep the normative spec in Ch4.

Reply:
agree with Pankaj, it's better to remove the hardcoded value from the requirments and just maintain the central table (and add "as per table" to the req text)

Reply:
Fixed in d7be782